code/modules/urist/mob/humanhostiles.dm

```diff
@@ -0,0 +1,152 @@
+/mob/living/simple_animal/hostile/urist
+	name = "gunman"
+	desc = "Armed and dangerous."
+	icon = 'icons/uristmob/simpleanimals.dmi'
+	icon_state = "gunman"
+	icon_living = "gunman"
+	icon_dead = "gunman_dead"
+	icon_gib = "syndicate_gib"
+	speak_chance = 0
+	turns_per_move = 5
+	response_help = "pokes"
+	response_disarm = "shoves"
+	response_harm = "hits"
+	speed = 12
+	stop_automated_movement_when_pulled = 0
+	maxHealth = 50
+	health = 50
+	harm_intent_damage = 5
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	attacktext = "punched"
+	a_intent = "harm"
+	ranged = 1
+	min_oxy = 5
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 1
+	min_co2 = 0
+	max_co2 = 5
+	min_n2 = 0
+	max_n2 = 0
+	unsuitable_atoms_damage = 15
+	faction = "neutral"
+	status_flags = CANPUSH
+	stat_attack = 1
+	projectiletype = /obj/item/projectile/bullet
+	projectilesound = 'sound/weapons/Gunshot.ogg'
+	casingtype = /obj/item/ammo_casing/c9mm
+	move_to_delay = 4
+	attack_sound = 'sound/weapons/punch3.ogg'
+	projectiletype = /obj/item/projectile/bullet/pistol
+
+/mob/living/simple_animal/hostile/urist/gunman //mostly redundant, for ease of spawning
+	minimum_distance = 4
+	retreat_distance = 2
+	faction = "syndicate"
+
+/mob/living/simple_animal/hostile/urist/commando //literally a clone of syndies with my speed tweaks and NT faction, adminfuckery purposes
+	faction = "NTIS"
+	name = "\improper NTIS Commando"
+	desc = "A henchman of the Internal Security department. You suddenly get an unpleasant sensation that you <I>'know too much'</I>."
+	ranged = 1
+	ranged_cooldown_cap = 5
+	rapid = 1
+	icon_state = "syndicateranged"
+	icon_living = "syndicateranged"
```

Comment:
It looks like one, yes. As per comment, it's essentially a clone of syndicate/ranged with the faction and move_to_delay edited because people use ranged syndies as NT response in CentCom.